src/components/Main/Modal/FileModal/FileModalContentFooter.vue

```diff
@@ -1,26 +1,28 @@
 <template>
   <div :class="$style.container" :data-is-white="isWhite">
-    <div :class="$style.channelPath">
-      {{ channelPath }}
-    </div>
+    <div :class="[$style.channelPath]" @click="onClick">#{{ channelPath }}</div>
```

Comment:
```suggestion
    <div :class="$style.channelPath" @click="onClick">#{{ channelPath }}</div>
```
一つしかないときは配列じゃなくてそのまま渡せたりする :+1: